kernel/Kconfig

```diff
@@ -651,6 +651,17 @@ config STACK_POINTER_RANDOM
 	  This is currently only implemented for systems whose stack pointers
 	  grow towards lower memory addresses.
 
+config BOUNDS_CHECK_BYPASS_MITIGATION
```

Comment:
Could call it `MITIGATE_BOUNDS_CHECK_BYPASS` maybe, to make it slightly shorter and less ambiguous.